src/test/java/io/cdap/plugin/servicenow/sink/ServiceNowSinkConfigTest.java

```diff
@@ -242,8 +239,12 @@ public void testCheckCompatibilityWithLogicalType() {
     Schema providedSchema = Schema.recordOf("providedSchema",
                                             Schema.Field.of("CreatedDateTime", Schema.of(Schema.LogicalType.
                                                                                            DATETIME)));
-    config.checkCompatibility(actualSchema, providedSchema, collector);
-    Assert.assertEquals(1, collector.getValidationFailures().size());
+    try {
+      config.checkCompatibility(actualSchema, providedSchema, collector);
```

Comment:
Assert.fail() if exception is not thrown

Reply:
Added